hawkular-openshift-agent/hawkular-openshift-agent.yaml

```diff
@@ -0,0 +1,119 @@
+id: hawkular-openshift-agent
+kind: Template
+apiVersion: v1
+name: Hawkular OpenShift Agent Template
+metadata:
+  name: hawkular-openshift-agent
+  metrics-infra: agent
+parameters:
+- description: The version of the image to use
+  name: IMAGE_TAG
```

Comment:
We have been using version since inception even though as you say tag is more correct.  I'm looking for consistency here